backend/src/test/kotlin/blue/mild/covid/vaxx/api/ApiTest.kt

```diff
@@ -0,0 +1,28 @@
+package blue.mild.covid.vaxx.api
+
+import blue.mild.covid.vaxx.utils.logResults
+import kotlinx.coroutines.runBlocking
+
+private const val TARGET_HOST = "https://covid-vaxx.stg.mild.blue"
```

Comment:
Jinak tohle by mohlo byt konfigurovatelne z envu

Reply:
to bych si myslel, ze je jedno, kdyz je to test..